### PR TITLE
Retrieve version number from SwiftCovFramework's Info.plist in version command

### PR DESCRIPTION
**Pull request by ikesyo**
This is the same way as Carthage.

### CODE REVIEW

**Comment by kishikawakatsumi** [comment]
It looks nice, thanks!

File: Source/swiftcov/Version.swift [hunk 1]
```diff
@@ -8,8 +8,7 @@
 
 import Commandant
 import Result
-
-private let version = "0.1.0"
+import let SwiftCovFramework.SwiftCovFrameworkBundleIdentifier
```
Comment (jpsim) [inline Source/swiftcov/Version.swift]: I wasn't aware it was possible to import a single declaration. Cool!

**Comment by segiddins** [comment]
:+1:

**Comment by ikesyo** [comment]
Thank you! :smiley:

**Pull request merged**